src/Assets/ExtensionInformation.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
A good thing about the version of the json we were using before is that it was strictly a subset of the full json. Can we try to make this json match + the extra extension info we're including here? If we make sure it matches, we can make this as well? If we make sure it matches, we can make a script to convert it to just the parts we need (I've been meaning to do that).

Also, if we're putting this here, we should remove the existing version https://github.com/microsoft/devhome/blob/93ab5ad9bac7f0d30ad0aa67b4b18362bf89e021/tools/ExtensionLibrary/DevHome.ExtensionLibrary/Assets/extensionResult.json

Reply:
I re-added the productIds array and added PublisherName and  ProductTitle's in english. We'll now use WinGet to get this information but will fallback to these english strings if WinGet is unavailable. I think we're good for now with this. I also removed the existing one you linked